### PR TITLE
Made some minor changes to Pipeline Installation

### PR DESCRIPTION
**Pull request by nkakkar81**
Made some minor changes to Pipeline Installation, and fixed two broken links:
1. Removed 'hello world'
2. Fixed Broken link to Using Pipeline Documentation
3. Modified Pre-reqs in Installation procedure.
4. Removed CRC specific instructions in Deploying application

### CODE REVIEW

**Review by nebelsprachen — APPROVED** [review]

File: modules/ROOT/pages/proc_deploying-an-application-to-openshift.adoc [hunk 1]
```diff
@@ -8,38 +8,17 @@
 [discrete]
 === Prerequisites
 
+* You have access to OpenShift Container Platform cluster.
```
Comment (Preeticp) [inline modules/ROOT/pages/proc_deploying-an-application-to-openshift.adoc:11]: Considering that we are removing the entire CRC section, please add a line similar to the one in the installing Pipelines operator section

Comment (Preeticp) [reply modules/ROOT/pages/proc_deploying-an-application-to-openshift.adoc:11]: Could you please fix the link on line 12 too? Does not work for me.

File: modules/ROOT/pages/proc_installing-pipelines-operator-in-web-console.adoc [hunk 2]
```diff
@@ -10,11 +10,10 @@ OpenShift Pipelines can be installed by using the operator listed in the OpenShi
 [discrete]
 === Prerequisites
 
-* An OpenShift 4 cluster is required for installation. Visit link:https://try.openshift.com/[try.openshift.com] for information on setting up an OpenShift 4 cluster.
+* Access to a running link:https://try.openshift.com/[OpenShift Container Platform 4] cluster with administrative privileges. Developers can use link:https://access.redhat.com/documentation/en-us/red_hat_codeready_containers/1.0/html/getting_started_guide/index[CodeReady Containers(CRC)] to deploy a local OpenShift Container Platform cluster quickly.
```
Comment (Preeticp) [inline modules/ROOT/pages/proc_installing-pipelines-operator-in-web-console.adoc:13]: remove 'to' - Access a running...
Space before (CRC)

**Review by Preeticp — COMMENTED** [review]
Overall, looks good. Some minor suggestions and it would be good to merge.

**Comment by Preeticp** [comment]
Closing this PR, in favor of a new consolidated PR from Nidhi.

**Pull request closed without merging**